### PR TITLE
Update copy for home page

### PR DESCRIPTION
**Pull request by ghost**
Types of changes
----------------

- New feature (non-breaking change which adds functionality)

Description
-----------

Improves copy on home page. New copy aims for better continuity, clarity in project details, fewer typos.

The previous copy had some typos and didn't flow as nicely. Didn't get scrAPD's value prop through as well.

Yes, it works on my local instance.

Checklist
---------

-  [] I have updated the documentation accordingly
-  [] I have written unit tests

(Not applicable)

Fixes scrapd/scrapdviz#

### CODE REVIEW

File: components/scrapd-home-view/component.js [hunk 1]
```diff
@@ -5,19 +5,24 @@ const ScrapdHomeView = () => {
   const intro = [
     <div key="0">
       <p>
-        ScrAPD is a collection of tools to automatically collect information about traffic fatalities in the city of
-        Austin, TX and render it on a modern dashboard.
+        ScrAPD is a web scraper that continuously and automatically collects traffic fatality information for the city
```
Comment (rgreinho) [inline components/scrapd-home-view/component.js]: ScrAPD is not simply a scraper, it is a collection of 4 tools: the scraper, the pipelines, the data sets, and the dashboard.

File: components/scrapd-home-view/component.js [hunk 2]
```diff
@@ -5,19 +5,24 @@ const ScrapdHomeView = () => {
...
+        of Austin, TX. The aggregated data is then made available for easy viewing in our scrAPD dashboard.
       </p>
       <p>
-        ScrAPD provides information to help understand traffic fatality patterns and empower decision makers to
-        implement solutions to prevent them.
+        ScrAPD’s constantly updated dashboard makes it easy for individuals and organizations lobbying towards a safer
```
Comment (rgreinho) [inline components/scrapd-home-view/component.js]: `advocating` sounds more positive than `lobbying`.

File: components/scrapd-home-view/component.js [hunk 3]
```diff
@@ -29,41 +34,40 @@ const ScrapdHomeView = () => {
   const grabData = [
     <div key="0">
       <p>
-        We are fully transparent with our data sets. We publish them on{' '}
+        The scrAPD web scraper generates data sets which populate our scrAPD dashboard. Use the “JSON” and “CSV” buttons
```
Comment (rgreinho) [inline components/scrapd-home-view/component.js]: We need to be consistent and write `ScrAPD` with the same casing everywhere.

File: components/scrapd-home-view/component.js [hunk 4]
```diff
@@ -29,41 +34,40 @@ const ScrapdHomeView = () => {
...
         You can find the scraping tool we use to retrieve data at this{' '}
-        <a href="https://github.com/scrapd/scrapd"> link</a>. and the source code for this dashboard{' '}
+        <a href="https://github.com/scrapd/scrapd"> link</a>. and the source code for the dashboard{' '}
         <a href="https://github.com/scrapdviz/scrapdviz"> here</a>.
       </p>
       <p>
-        ScrAPD is a project developed mostly by vonlunteers from Open Austin, a local organization solving civic
-        problems using design and technology. You can learn more about it on their{' '}
-        <a href="https://open-austin.org">website</a>.
+        ScrAPD is a project developed mostly by volunteers from Open Austin, a local organization solving civic problems
```
Comment (rgreinho) [inline components/scrapd-home-view/component.js]: > ScrAPD is a project developed mostly by volunteers from Open Austin, a local organization solving civic problems

should be:

ScrAPD is a project developed by volunteers, mostly from Open Austin, a local organization solving civic problems

File: components/scrapd-home-view/component.js [hunk 5]
```diff
@@ -72,9 +76,10 @@ const ScrapdHomeView = () => {
     <div>
       <ScrapdHero
         title="scrAPD"
-        subtitle1="Safety data, collected automatically for you"
-        subtitle2="ScrAPD scrapes the web for traffic fatality data, making it accessible to all"
-        url="https://res.cloudinary.com/ab91/image/upload/v1567305230/scrAPD%20Viz/hero_image.png"
+        subtitle1="Austin’s latest traffic fatality data, in one spot"
```
Comment (rgreinho) [inline components/scrapd-home-view/component.js]: Maybe we can have a tagline a tad less positive. We are talking about deaths.

**Review by rgreinho — CHANGES_REQUESTED** [review]
Great job @tom-nook! This is definitely an improvement compared to what we had before.

I just gave a few remarks that need to be addressed.

File: components/scrapd-home-view/component.js (hunk 2 above)
Comment (tom-nook) [reply components/scrapd-home-view/component.js]: Complete

File: components/scrapd-home-view/component.js (hunk 3 above)
Comment (tom-nook) [reply components/scrapd-home-view/component.js]: Complete

File: components/scrapd-home-view/component.js (hunk 4 above)
Comment (tom-nook) [reply components/scrapd-home-view/component.js]: Complete

File: components/scrapd-home-view/component.js (hunk 5 above)
Comment (rgreinho) [reply components/scrapd-home-view/component.js]: What do you think of: "Visualize Austin traffic fatalities from another angle"?

This is the tagline we use for the repo.

File: components/scrapd-home-view/component.js (hunk 1 above)
Comment (tom-nook) [reply components/scrapd-home-view/component.js]: Is it inappropriate do you know to use `web scraper` in the context of this sentence, in the section on Data sets?

>The ScrAPD web scraper generates data sets which populate our ScrAPD dashboard.

File: components/scrapd-home-view/component.js (hunk 5 above)
Comment (tom-nook) [reply components/scrapd-home-view/component.js]: Complete

Comment (tom-nook) [reply components/scrapd-home-view/component.js]: Changed to "Visualize Austin traffic fatalities from another angle"

File: components/scrapd-home-view/component.js (hunk 1 above)
Comment (tom-nook) [reply components/scrapd-home-view/component.js]: Complete!

**Review by rgreinho — APPROVED** [review]

**Comment by rgreinho** [comment]
Fixes #165.

**Pull request merged**